static-checks/rule-identifiers/main.fmf

```diff
@@ -0,0 +1,10 @@
+summary: Search for policy references in rules
+test: python3 -m lib.runtest ./test.py
+result: custom
+environment+:
+    PYTHONPATH: ../..
+duration: 10m
+adjust:
+  - when: distro == rhel-10
+    enabled: false
+    because: rhel10 profiles are in draft state with missing references
```

Comment:
Wouldn't waivers be better here, to not forget about this as RHEL-10 develops further?

Reply:
Honestly, not sure if waivers would be better as it doesn't change much in forgetting about it. (supposing our current usual workflow - waiver, upstream issue, and forget).

Still I'll think about it. But as RHEL10 are still deeply in draft state, those waiver+upstream issue would be without `productization-issue` label and thus easily lost in issue backlog.